xfce/libxfce4util/Portfile

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem 1.0
+PortGroup       legacysupport 1.1
```

Comment:
It may be `legacysupport.newest_darwin_requires_legacy 10`, however I did not test it beyond 10.6, so not sure if later ones need anything from legacysupport.

Reply:
Added.